pkgs/tools/security/dumpasn1/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "dumpasn1";
+  version = "unstable-2014-06-26";
+
+  hardeningDisable = [ "all" ];
```

I wouldn't disable everything, instead just disable the specific error and give a reason:
```suggestion
  # uses constant for indenting
  NIX_CFLAGS_COMPILE = "-Wno-error=format-security";
```